srf_generation/source_parameter_generation/common.py

```diff
@@ -1,6 +1,7 @@
 from os.path import abspath, join, dirname
 
 import pandas as pd
+from numpy import digitize
```

Comment:
why not import numpy like np?

Reply:
It only needs digitize and asarray, I suppose it wouldn't hurt or help to change it. I can if you want

Reply:
Yeah. I don't think it'll hurt performance. It'll just make it a little clearer that these are numpy functions.